doc/adrs/032_relax_our_constraints_on_some_of_our_identifiers.adoc

```diff
@@ -28,7 +27,22 @@ WIP
 
 == Consequences
 
-Projects consuming Sirius Components such as Sirius Web will need to be updated in order to explicitely decide what they will use as identifiers.
+Projects consuming Sirius Components such as Sirius Web will need to be updated in order to explicitly decide what they will use as identifiers.
 Entities in Sirius Web will still use UUID and the transformation from UUID to Strings will be made by Sirius Web code.
 For the moment, we recommend keeping UUID saved as Strings just like in Sirius Web since it is our nominal use case.
 Only advanced projects with complex needs should try to setup their own complex identifiers.
+
+When the `Diagram.getId()` become a `String`, this involves a type mismatch with `Node.getId()` and `Edge.getId()` that are still `UUID`.
+A `diagramElementId` can no longer refers to a diagramId. A `diagramElement` is now exclusively a part of a diagram (node, edge, label...).
```

Comment:
we could also change the type of `Node#id` and `Edge#id` on the backend and not have all those impact

Reply:
Rewritten with the fact that `Node#id`, `Edge#id` and `LayoutData#id` and `Label#id` is now a `String`.